Api/Services/VisitService.cs

```diff
@@ -1,8 +1,87 @@
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.OpenApi.Extensions;
+using Reservant.Api.Data;
+using Reservant.Api.Models;
+using Reservant.Api.Models.Dtos.Visit;
+using Reservant.Api.Validation;
+using Reservant.Api.Validators;
+
 namespace Reservant.Api.Services;
 
 /// <summary>
 /// Service for managing visits
 /// </summary>
-public class VisitService
+public class VisitService(
+    ApiDbContext context,
+    UserManager<User> userManager,
+    ValidationService validationService
+    )
 {
+
+    public async Task<Result<VisitSummaryVM>> CreateVisitAsync(CreateVisitRequest request, User user)
+    {
+        var restaurant = await context.Restaurants.FindAsync(request.RestaurantId);
```

Comment:
to sprawdzenie trzeba przenieść w walidator stolika (chociaż jeśli po prostu sprawdzisz czy stolik o takich RestaurantId i TableId istnieje, to to automatycznie będzie znaczyło że restauracja też istnieje)